modules/test-runner/src/util/bn.ts

```diff
@@ -3,3 +3,7 @@ import { BigNumber, bigNumberify, BigNumberish, formatEther, parseEther } from "
 export const toWei = (n: BigNumberish): BigNumber => parseEther(n.toString());
 
 export const inverse = (bn: BigNumberish): string => formatEther(toWei(toWei("1")).div(toWei(bn)));
+
+export const calculateExchange = (amount: BigNumber, swapRate: string): BigNumber => {
+  return bigNumberify(formatEther(amount.mul(parseEther(swapRate))).replace(/\.[0-9]*$/, ""));
+};
```

Comment:
isn't this a function exported from the client? seems like we should use that (but don't have a huge pref here, i think we just have the same fn in three places: here, client, daicard)

Reply:
not sure here just used what was available before in the `swap.test.ts` cc @rhlsthrm